src/Map.ts

```diff
@@ -639,14 +649,14 @@ export class Map extends maplibregl.Map {
     let terrainEventTriggered = false;
     let terrainEventData: LoadWithTerrainEvent;
 
-    this.once("ready", () => {
+    void this.once("ready", () => {
```

Comment:
is this `void` return type needed unless we porposely want to discard a return value here, no? I'm presuming it just returns `this` which is harmless to ignore. Was the linter complaining? Otherwise for consistency we should do it for _every_ void function call. Which is a lot.

Reply:
same with the others below

Reply:
Just saw the lint error above. That's a strange lint rule to have in there.

Reply:
There's a lot of lint errors around the controllers which when you fix one of them there's another what appears, I dont know what to priotirize, but there's no return in this function so I think it's good to remove a few lines of warnings